code/global/TTTEvent.cs

```diff
@@ -0,0 +1,49 @@
+using Sandbox;
+
+namespace TTTReborn.Globals
+{
+    public static class TTTEvent
+    {
+        public static class Player
+        {
+            public const string Died = "tttreborn.player.died";
+            public const string InitialSpawn = "tttreborn.player.initialspawn";
+            public const string Spawned = "tttreborn.player.spawned";
+            public const string TakeDamage = "tttreborn.player.takedamage";
+
+            public static class CarriableItem
+            {
+                public const string Drop = "tttreborn.player.carriableitem.drop";
+                public const string PickUp = "tttreborn.player.carriableitem.pickup";
+            }
+
+            public static class Inventory
+            {
+                public const string Clear = "tttreborn.player.inventory.clear";
+            }
+
+            public static class Role
+            {
+                public const string OnSelect = "tttreborn.player.role.onselect";
+            }
+
+            public static class Spectating
+            {
+                public const string Change = "tttreborn.player.spectating.change";
+            }
+        }
+
+        public static class Settings
+        {
+            public static class Instance
```

Comment:
Feels like an extra step for no reason. The others devs may disagree.

Reply:
Yeah probably can get rid of instance as well.